lib/intergallactic/transaction.js

```diff
@@ -24,7 +33,7 @@ const config = require('../config/constant'),
  * };
  * const opt = { txnType: 1 // required }
  * const newTxn = new igc.Transaction(txn, opt);
- * newTxn.signNSend(privateKey)
+ * newTxn.signNBroadcast(privKey)
```

Comment:
I think better to broadcast a signed tx. Sign the broadcast. More secure.

Reply:
you mean to change the naming from signNBroadcast to broadcastSignedTx?

Reply:
It's better to not pass private_key between objects. You can have a `signer` object to just sign transactions. This object can hold the private_key.

Reply:
I agree.. In this case my `signer` would be the `signByPrivKey` function.